src/SparqlLuaQueryRunner.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\SPARQL;
+
+use MediaWiki\Http\HttpRequestFactory;
+
+class SparqlLuaQueryRunner {
+
+	private HttpRequestFactory $requestFactory;
+	private string $sparqlEndpoint;
+
+	/**
+	 * @param HttpRequestFactory $requestFactory
+	 * @param string $sparqlEndpoint
+	 */
+	public function __construct( HttpRequestFactory $requestFactory, string $sparqlEndpoint ) {
+		$this->requestFactory = $requestFactory;
+		$this->sparqlEndpoint = $sparqlEndpoint;
```

Comment:
In PHP 8 you don't need to manually define and set variables like this. See: https://php.watch/versions/8.0/constructor-property-promotion